File: ecl/hybris-ecl-provider/hybris-ecl-provider/Hybris API/CatalogVersion.cs
namespace SDL.ECommerce.Hybris.API.Model
{
    using System;
    using System.Collections.Generic;
    using System.Diagnostics.CodeAnalysis;

    /// <summary>
    /// Catalog version
    /// </summary>
    [SuppressMessage("StyleCop.CSharp.NamingRules", "SA1300:ElementMustBeginWithUpperCaseLetter", Justification = "auto generated class needs to map to json file")]
    public class CatalogVersion
    {
        public string id { get; set; }
        public DateTime lastModified { get; set; }
        public string url { get; set; }
        public List<Category> categories { get; set; }
    }
}

File: connectors/hybris-ecommerce-connector/src/main/java/com/sdl/ecommerce/hybris/api/model/CatalogVersion.java
package com.sdl.ecommerce.hybris.api.model;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.Date;
import java.util.List;

/**
 * CatalogVersion
 *
 * @author nic
 */
public class CatalogVersion {

    @JsonProperty("id")
    private String id;

    @JsonProperty("lastModified")
    private Date lastModified;

    @JsonProperty("url")
    private String url;

    @JsonProperty("categories")
    private List<Category> categories;

    public List<Category> getCategories() {
        return categories;
    }

    public String getId() {
        return id;
    }

    public Date getLastModified() {
        return lastModified;
    }

    public String getUrl() {
        return url;
    }
}
